code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -0,0 +1,67 @@
+/obj/item/gun/ballistic/bow
+	name = "wooden bow"
+	desc = "some sort of primitive projectile weapon. used to fire arrows."
+	icon_state = "bow"
+	item_state = "bow"
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY //need both hands to fire
+	force = 5
+	mag_type = /obj/item/ammo_box/magazine/internal/bow
+	fire_sound = 'sound/weapons/bowfire.ogg'
+	slot_flags = ITEM_SLOT_BACK
+	item_flags = NONE
+	casing_ejector = FALSE
+	internal_magazine = TRUE
+	pin = null
+	no_pin_required = TRUE
+	trigger_guard = TRIGGER_GUARD_NONE //so ashwalkers can use it
```

Comment:
```suggestion
	trigger_guard = TRIGGER_GUARD_ALLOW_ALL //so ashwalkers can use it
```

Reply:
that can be done thank you for the suggestion